daemon/logger/awslogs/cwlogsiface_mock_test.go

```diff
@@ -49,6 +60,13 @@ func (m *mockcwlogsclient) PutLogEvents(input *cloudwatchlogs.PutLogEventsInput)
 	return output.successResult, output.errorResult
 }
 
+func (m *mockcwlogsclient) CreateLogGroup(input *cloudwatchlogs.CreateLogGroupInput) (*cloudwatchlogs.CreateLogGroupOutput, error) {
+	// m.createLogGroupArgument <- input
```

Comment:
@samuelkarp I agree on all suggestions. Didn't know where to adjust the unit test expectations, so I just commented out for now or else the unit test hangs on this.